deployments/kube-local-config.yml

```diff
@@ -1,13 +1,13 @@
 apiVersion: v1
 data:
-  LDAP_ADMIN_GROUPBASE: cn=DL_ADMIN_TEAM,OU=GLOBAL,ou=Groups,dc=kubi,dc=ca-gip,dc=github,dc=com
-  LDAP_ADMIN_USERBASE: dc=kubi,dc=ca-gip,dc=github,dc=com
-  LDAP_BINDDN: cn=admin,dc=kubi,dc=ca-gip,dc=github,dc=com
-  LDAP_GROUPBASE: ou=LOCAL,ou=Groups,dc=kubi,dc=ca-gip,dc=github,dc=com
+  LDAP_ADMIN_GROUPBASE: cn=DL_ADMIN_TEAM,OU=GLOBAL,ou=Groups,dc=kubi,dc=cagip,dc=github,dc=com
```

Comment:
Why the change?

Reply:
the "-" is no longer supported (otherwise the pod will not be started)